Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -210,20 +228,24 @@ function Enable-SecureFTPDeploymentForAppServices
 
     if ([String]::IsNullOrWhiteSpace($context))
     {
-        Write-Host "No active Azure login session found. Exiting..." -ForegroundColor $([Constants]::MessageType.Error)
-        return
+        Write-Host "Connecting to Azure account..." -ForegroundColor $([Constants]::MessageType.Info)
+        Write-Host $([Constants]::SingleDashLine)
+        Connect-AzAccount -Subscription $SubscriptionId -ErrorAction Stop | Out-Null
+        Write-Host "Connected to Azure account." -ForegroundColor $([Constants]::MessageType.Update)
+        Write-Host $([Constants]::SingleDashLine)
     }
 
     # Setting up context for the current Subscription.
     $context = Set-AzContext -SubscriptionId $SubscriptionId -ErrorAction Stop
     
-    
+    if(-not($AutoRemediation))
+    {
     Write-Host "Subscription Name: [$($context.Subscription.Name)]"
```

Comment:
Indentation still looks incorrect to me.

Reply:
worked on that part